Packs/CarbonBlackDefense/Integrations/CarbonBlackLiveResponseCloud/test_data/test.txt

```diff
@@ -0,0 +1,2 @@
+test data
+test data
```

Comment:
Do we need this file?

Reply:
Yes 
need for Unit Test for open real file